doc/doc/basics/ddlog.md

```diff
@@ -5,7 +5,7 @@ layout: default
 # Writing Applications in DDlog
 
 DDlog is a higher-level language for writing DeepDive applications in succinct Datalog-like syntax.
-We are gradually extending the language to allow expression of advanced SQL queries used by complex extractors as well as a variety of factors with Markov Logic-like syntax.
+We are gradually extending the language to allow expression of advanced SQL queries used by complex extractors as well as a variety of factors with Markov Logic-like syntax. A reference for ddlog lanugage features can be found [here](https://github.com/HazyResearch/ddlog/wiki/DDlog-Language-Features).
```

Comment:
Sorry for being picky here, but I think [keeping one sentence per line](https://news.ycombinator.com/item?id=4642395) makes it easy to see what's changed.